.github/workflows/prettier.yml

```diff
@@ -0,0 +1,35 @@
+name: Check formatting
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: "*"
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install Prettier
+        run: npm install -g prettier

```

Comment:
ah we also need to match the package.json.prettier config as I've opted out of a v3 prettier convention